shavispackage/src/shavissrc/shavis.py

```diff
@@ -491,12 +491,17 @@ def start():
 	sha256_dv = [int(i, 16) for i in sha256] # dv  = Decimal Values         |   hex -> decimal -> list
 	sha256_dvm = []		 					 # dvm = Decimal Values Matrix  |
 	for i in range(0, 8): sha256_dvm.append(sha256_dv[i*8:(i+1)*8]) #       |   list[64] -> matrix[8][8]
+ 
 	# [==================[Theme Loader]==================]
-	theme = theme_dir / (theme + ".hex") # insert theme name
-	with open(theme) as file:            # open theme file
+ 
+	theme_path = f"./conf/themes/{theme}.hex" # insert theme name
```

Comment:
this one is supposed to be left as `theme = theme_dir / (theme + ".hex")`, otherwise it wont work when downloaded off of PyPI.

Reply:
Because pathlib is used to derive the path on the host machine, the path can't be explicitly declared and must be derived through pathlib.